test/OrchardCore.Tests/ContentManagement/ContentElementTests.cs

```diff
@@ -66,4 +66,30 @@ public void Get_WhenCastingConcreteTypeThenBaseType_ReturnNewInstance()
         // contentPart should be returned from cache, so it must be same as actualPart.
         Assert.Same(contentPart, actualPart);
     }
+
+    [Theory]
+    [InlineData(15)]
+    [InlineData(-15)]
+    [InlineData(null)]
+    public void Apply_WhenCalledWithNullProperty_SetThePropertyToNull(int? minutes)
+    {
+        // Arrange
+        var contentItem = new ContentItem();
+
+        // Act
+        contentItem.Apply(new TestContentPart()
+        {
+            Minutes = minutes,
+        });
+
+        var part = contentItem.As<TestContentPart>();
+
+        // Assert
+        Assert.Equal(minutes, part.Minutes);
```

Comment:
@hishamco this test is now useless. I think you missed understanding the test that I put together. Your version of the test will pass even if you apply it to the main branch. The test I had before your commit tests to see if the object was updated from a value back to null. So please revert the last commit

Reply:
Wait a minute let me double check again

Reply:
I just understand what you mean, seems no need for the `Theory` here, so I will revert the test changes